appengine/php72/getting-started/test/CloudSqlTest.php

```
@@ -30,7 +30,14 @@ public function setUp()
         $dbUser = $this->requireEnv('CLOUDSQL_USER');
         $dbPass = $this->requireEnv('CLOUDSQL_PASSWORD');
         $dbName = getenv('CLOUDSQL_DATABASE_NAME') ?: 'bookshelf';
-        $dsn = "mysql:unix_socket=/cloudsql/${connection};dbname=${dbName}";
+        $socket = "/cloudsql/${connection}";
+
+        if (!file_exists($socket)) {
+            $this->markTestSkipped(
```

Why not fail instead of skipping?

---

Because I didn't want to try setting up the `cloud_sql_proxy` on Kokoro yet. It's not a great excuse, you caught me.

---

I would like to get the `cloud_sql_proxy` running in another PR, as this will take more consideration to get right.